src/index.js

```diff
@@ -23,8 +26,22 @@ class CreateDynamoDBGlobalTables {
     const region = provider.getRegion();
     const tableNames = this.getTableNames();
 
-    await Promise.all(tableNames.map(t => this.createGlobalTable(t, region)));
-    await Promise.all(tableNames.map(t => this.addReplica(t, region)));
+    if (region !== 'ca-central-1') {
+      await Promise.all(tableNames.map(t => this.createGlobalTable(t, region)));
```

Comment:
DUPLICATION ALERT! looks like the first operation that you do while calling these functions is
 `const tableHasBeenUpgraded = await this.tableIsAtVersionNumber(tableName);`

you can do it first for all the regions and then only run `createGlobalTable` and `addReplica` for regions which had not been upgraded

Reply:
shouldn't `createGlobalTable` be only called once? and then the other regions be added through `addReplica`? `dynamo.createGlobalTable` vs `dynamo.updateGlobalTable`

Reply:
Oh, there is definitely code duplication on this spike, I didn't really are about that for this hack tbh.

This code was already here so the logic hasn't changed.